pytorch/training/docker/1.8.0/py3/example/Dockerfile.gpu

```diff
@@ -0,0 +1,12 @@
+
+# Use the Deep Learning Container as a base Image
+ARG BASE_IMAGE=""
+
+FROM $BASE_IMAGE
+
+RUN pip install tensorboardX==1.6.0
```

Comment:
Latest version of tensorboardX available https://pypi.org/project/tensorboardX/2.1/
Should we upgrade?

Reply:
Let's keep it now for stability. Could update it later on.